test/extended/images/mongodb_replica.go

```diff
@@ -0,0 +1,126 @@
+package images
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+)
+
+var _ = g.Describe("[images][mongodb] openshift mongodb replication", func() {
+	defer g.GinkgoRecover()
+
+	const (
+		templatePath         = "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json"
+		deploymentConfigName = "mongodb"
+		insertCmd            = `db.bar.save({ "status": "passed" })`
+	)
+
+	const (
+		expectedReplicasAfterDeployment = 3
```

Comment:
I still not sure that it worth it... Adding more code that will be executed each time with test and all this just for a rare case when someone should update a number in 2 places?